Dependencies/xr/Source/OpenXR/XR.cpp

```diff
@@ -75,14 +26,61 @@ namespace xr
         };
     }
 
+    XrSessionContext XrRegistry::s_context{};
+
+    struct XrSessionContext::Impl
+    {
+        XrSessionContext::Impl()
+            : extensions(std::make_unique<XrSupportedExtensions>()) {}
+
+        void PopulateExtensions()
+        {
+            if (instance.Get() == XR_NULL_HANDLE)
+            {
+                throw std::exception(/*Attempted to populate extensions when instance was null*/);
+            }
+
+            extensions->PopulateDispatchTable(instance.Get());
+        }
+
+        XrTime GetDisplayTime()
+        {
+            return displayTime;
+        }
+
+        InstanceHandle instance;
```

Comment:
nit: Stylistically, the convention in the broader Babylon Native system is to capitalize public struct members (and use `m_` for private members). I'm fine with this being left for a later clang format to pick up on, though; it's also fine for the XR dependency to establish and use different conventions.

Reply:
this should now be fixed